packages/flutter/test_fixes/material.dart

```diff
@@ -554,4 +554,20 @@ void main() {
 
   // Change made in https://github.com/flutter/flutter/pull/100381
   TextSelectionOverlay.fadeDuration;
+
+  // Changes made in https://github.com/flutter/flutter/pull/97972
+  ThemeData themeData = ThemeData();
+  themeData = ThemeData(toggleableActiveColor: Colors.black);
+  themeData = ThemeData(
+    toggleableActiveColor: Colors.black,
+  );
+  themeData = themeData.raw(toggleableActiveColor: Colors.black);
+  themeData = themeData.raw(
```

Comment:
This is a constructor, in contrast to `copyWith`, which is a method.

```suggestion
  themeData = ThemeData.raw(toggleableActiveColor: Colors.black);
  themeData = ThemeData.raw(
```

Analysis is disabled for this file because it is intentionally full of broken code. It may be helpful to put the 'before' code under normal analysis to catch typos like this.

Reply:
That's my bad,  I didn't realize it, thank you